lib/kerosene/html.ex

```diff
@@ -150,4 +149,4 @@ defmodule Kerosene.HTML do
     opts = Keyword.merge(opts, [theme: theme])
     Keyword.merge(@default, opts)
   end

```

Comment:
I've originally refactored this function to 
```
defp render_page_list(page_list, paginator, [theme: theme, path: _path, params: _params, opts: _opts]) do
    case theme do
      :bootstrap  -> Kerosene.HTML.Boostrap.generate_links(page_list, paginator)
      :bootstrap4 -> Kerosene.HTML.Boostrap4.generate_links(page_list, paginator)
      :foundation -> Kerosene.HTML.Foundation.generate_links(page_list, paginator)
      :semantic   -> Kerosene.HTML.Semantic.generate_links(page_list, paginator)
      _           -> generate_links(page_list, paginator)
    end
  end
```
and completely removed this one
```
  defp render_page_list(page_list, paginator, _opts) do
    generate_links(page_list, paginator)
  end
```
but looks like it's needed if `:theme` is not obligatory, so reverted changes back